Modules/Office365DSC/Examples/Resources/SPOHomeSite/1-ConfigureSPOHomeSite.ps1

```diff
@@ -0,0 +1,25 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $credsGlobalAdmin
+    )
+    Import-DscResource -ModuleName Office365DSC
+
+    node localhost
+    {
+        SPOHomeSite "ff4a977d-4d7d-4968-9238-2a1702aa699c"
```

Comment:
For the example's purpose, give the resource a meaningful name instead of a GUID.